src/controller/blob/controller.go

```diff
@@ -75,6 +73,15 @@ type Controller interface {
 
 	// GetAcceptedBlobSize returns the accepted size of stream upload blob.
 	GetAcceptedBlobSize(sessionID string) (int64, error)
+
+	// Touch updates the blob status and increase version every time.
+	Touch(ctx context.Context, blob *blob.Blob) (int64, error)
```

Comment:
Touch should only be called by the head middleware, to update the status to "None"?

Reply:
updated.